src/connection.c

```diff
@@ -353,14 +369,17 @@ static noPollConn * nopoll_tls_common_conn (noPollCtx  * ctx,char * serverAddr,c
             ParodusInfo("Connecting in Ipv6 mode\n");
             connection = nopoll_conn_tls_new6 (ctx, opts,serverAddr,serverPort,NULL,get_parodus_cfg()->webpa_path_url,NULL,NULL);
         } else {
```

Comment:
Update get_parodus_cfg()->flags to FLAGS_IPv6_IPv4

Reply:
To update get_parodus_cfg()->flags to FLAGS_IPv6_IPv4 , we need ParodusCfg struct in nopoll_tls_common_conn. is that really required to update get_parodus_cfg()->flags with FLAGS_IPv6_IPv4 ? if yes then we need to change the function prototype declaration for createNopollConnection and nopoll_tls_common_conn to pass ParodusCfg as new args.

Reply:
Not required